### PR TITLE
Update alluxio download URI in vagrant

### PR DESCRIPTION
**Pull request by chcn**

### CODE REVIEW

File: integration/vagrant/vagrant/config_software.rb [hunk 1]
```diff
@@ -202,8 +202,11 @@ def spark_profile
   end
 
   def alluxio_dist(alluxio_version)
-    hadoop_major_minor_version = @version.split('.')[0..1].join('.')
-    return "alluxio-#{alluxio_version}-hadoop-#{hadoop_major_minor_version}-bin.tar.gz"
+		if alluxio_version.start_with?("1")
```
Comment (ZacBlanco) [inline integration/vagrant/vagrant/config_software.rb]: why is this indented so much?

Comment (chcn) [reply integration/vagrant/vagrant/config_software.rb]: tab vs space issue, resolved

**Comment by cheng-chang** [comment]
alluxio-bot, merge this please

**Comment by cheng-chang** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please

**Pull request merged**